code/modules/mob/living/simple_animal/hostile/mecha_pilot.dm

```diff
@@ -292,6 +292,6 @@
 
 /mob/living/simple_animal/hostile/syndicate/mecha_pilot/Goto(target, delay, minimum_distance)
 	if(mecha)
-		SSmove_manager.move_to(mecha, target, minimum_distance, mecha.movedelay * mecha.step_multiplier)
+		SSmove_manager.move_to(mecha, target, minimum_distance, mecha.movedelay)
```

Comment:
The only real remaining use case of step_multiplier after the previous pr.

Gonna test that simple mobs respect the vehicles movement delay with the multiplier removed. That's really my only worry in this pr. 

Player mobs driving mechs seems completely fine